### PR TITLE
Job monitor simplification

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

CHANGE: Added VO selector to JobMonitor

ENDRELEASENOTES

### CODE REVIEW

File: src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js [hunk 1]
```diff
@@ -282,7 +251,7 @@ Ext.define("DIRAC.JobMonitor.classes.JobMonitor", {
       ["site", "site"],
       ["status", "status"],
       ["types", "jobType"],
-      ["OwnerGroup", "OwnerGroup"],
```
Comment (chrisburr) [inline src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js:285]: I'm not sure why `OwnerGroup` is being removed?

Comment (fstagni) [reply src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js:285]: It is replaced by VO

Comment (andresailer) [reply src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js:285]: But OwnerGroup is still important to separate within a VO?

Comment (fstagni) [reply src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js:285]: indeed, restored.

File: src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js [hunk 2]
```diff
@@ -727,7 +683,7 @@ Ext.define("DIRAC.JobMonitor.classes.JobMonitor", {
       mode: "local",
       store: new Ext.data.ArrayStore({
         fields: ["category"],
-        data: [["Status"], ["Site"], ["Minor Status"], ["Application Status"], ["Owner"], ["Owner Group"], ["Job Group"], ["Job Type"]],
+        data: [["Status"], ["Site"], ["Minor Status"], ["Application Status"], ["Owner"], ["VO"], ["Job Group"], ["Job Type"]],
```
Comment (andresailer) [inline src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js]: Should this still have "Owner Group" ? I don't know what this does at all.

Comment (aldbr) [inline src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js]: Should `Owner Group` be restored here too?

Comment (fstagni) [reply src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js]: Done.

**Review by aldbr — APPROVED** [review]

**Pull request merged**